### PR TITLE
Allow field tests to iterate through different configs

### PR DESCRIPTION
**Pull request by timleslie**
In particular, this will let us apply all of our tests to the different types of `Select` field that exist.

### CODE REVIEW

File: packages/fields-location-google/src/test-fixtures.js [hunk 1]
```diff
@@ -10,13 +10,11 @@ export const name = 'Location';
...
-export const exampleValue = 'ChIJOza7MD-uEmsRrf4t12uji6Y';
-export const exampleValue2 = 'ChIJ_9gDOjmuEmsRB7WRXm_Y6o8';
+export const exampleValue = () => 'ChIJOza7MD-uEmsRrf4t12uji6Y';
+export const exampleValue2 = () => 'ChIJ_9gDOjmuEmsRB7WRXm_Y6o8';
 export const fieldName = 'venue';
 export const subfieldName = 'googlePlaceID';
-export const fieldConfig = {
-  googleMapsKey: process.env.GOOGLE_API_KEY,
-};
+export const fieldConfig = () => ({ googleMapsKey: process.env.GOOGLE_API_KEY });
```
Comment (singhArmani) [inline packages/fields-location-google/src/test-fixtures.js:17]: should we have default key value for google map too? @timleslie

File: tests/api-tests/fields/filter.test.js [hunk 2]
```diff
@@ -8,7 +8,7 @@ const testModules = globby.sync(`packages/**/src/**/test-fixtures.js`, {
 });
 testModules.push(path.resolve('packages/fields/tests/test-fixtures.js'));
 
-multiAdapterRunners().map(({ runner, adapterName }) =>
+multiAdapterRunners('knex').map(({ runner, adapterName }) =>
```
Comment (singhArmani) [inline tests/api-tests/fields/filter.test.js:11]: do we need this only for `knex`?

**Review by singhArmani — CHANGES_REQUESTED** [review]

File: packages/fields-location-google/src/test-fixtures.js (hunk 1 above)
Comment (timleslie) [reply packages/fields-location-google/src/test-fixtures.js:17]: It's not required to get tests to run, so no need for it.

**Review by singhArmani — APPROVED** [review]
This looks good now 👍

**Pull request merged**